tests/unit/test_install.py

```diff
@@ -220,3 +220,55 @@ def test_choices_happy(mocker):
     mocker.patch("builtins.input", return_value="1")
     res = install._choice("foo", ["a", "b"])
     assert "b" == res
+
+
+def test_step_list(mocker):
+    ws = mocker.Mock()
+    install = Installer(ws)
+    from databricks.labs.ucx.framework.tasks import _TASKS, Task
+
+    _TASKS.clear()
```

Comment:
Unit tests may be run from multiple threads in a random order, so this one is not deterministic.

Reply:
Darn, I need to find a way to make this testable... in general, using protected fields seems to be complained about but some linters.